src/build/arg_group.rs

```diff
@@ -522,9 +520,9 @@ mod test {
             .requires_all(&["r2", "r3"])
             .requires("r4");
 
-        let args = vec!["a1".key(), "a4".key(), "a2".key(), "a3".key()];
-        let reqs = vec!["r1".key(), "r2".key(), "r3".key(), "r4".key()];
-        let confs = vec!["c1".key(), "c2".key(), "c3".key(), "c4".key()];
+        let args = vec!["a1".into(), "a4".into(), "a2".into(), "a3".into()];
+        let reqs = vec!["r1".into(), "r2".into(), "r3".into(), "r4".into()];
+        let confs = vec!["c1".into(), "c2".into(), "c3".into(), "c4".into()];
```

Comment:
You mean these asserts? I don't know, but why not?

Reply:
No, I meant improving the messages from `_debug_asserts()` in #1797

Reply:
Well, the main reason I originally made this was I wanted to be able to see inputs and outputs in debug messages. I've tried to fix several bugs, but the lack of names made me postpone it.

But yeah, this also makes _debug_asserts` improvements possible. I'll proceed to it as soon as I'm done with the stalled PR I promised to help with.